<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>601</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>734</width>
    <height>601</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>734</width>
    <height>601</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>216</width>
      <height>155</height>
     </rect>
    </property>
    <property name="minimumDate">
     <date>
      <year>2013</year>
      <month>9</month>
      <day>14</day>
     </date>
    </property>
    <property name="maximumDate">
     <date>
      <year>2017</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Monday</enum>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ЧелИнвестБанк</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Альтернатива</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Мастер-дом</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>МЖКО</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>501</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>216</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="summWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>0</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Чел:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Альт:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Мастер:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>МЖКО:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Общая:</string>
     </property>
    </item>
   </widget>
   <zorder>listWidget_2</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>comboBox</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>listWidget</zorder>
   <zorder>summWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchslot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarWidget</sender>
   <signal>selectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>changedpage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>dclickedfilelist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchslot()</slot>
  <slot>changedpage()</slot>
  <slot>dclickedfilelist()</slot>
 </slots>
</ui>
